tests/phpunit/unit/api/post-meta/beansPostMetaPageTemplateReload.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/** Tests for beans_post_meta_page_template_reload.
+ *
+ * @package Beans\Framework\Tests\Unit\API\Post_Meta
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Unit\API\Post_Meta;
+
+use Beans\Framework\Tests\Unit\Test_Case;
+use Brain\Monkey;
+
+/**
+ * Class Tests_Beans_Post_Meta_Page_Template_Reload
+ *
+ * @package Beans\Framework\Tests\Unit\API\Post_Meta
+ * @group   api
+ * @group   api-post-meta
+ */
+class Tests_Beans_Post_Meta_Page_Template_Reload extends Test_Case {
```

Comment:
Whoopsie, please remove the underscores between the function name words and smoosh together like this `Tests_BeansPostMetaPageTemplateReload`.